code/modules/admin/sql_message_system.dm

```diff
@@ -72,7 +78,14 @@
 					return
 				expiry = query_validate_expire_time.item[1]
 			qdel(query_validate_expire_time)
-	var/datum/DBQuery/query_create_message = SSdbcore.NewQuery("INSERT INTO [format_table_name("messages")] (type, targetckey, adminckey, text, timestamp, server, server_ip, server_port, round_id, secret, expire_timestamp) VALUES ('[type]', '[target_ckey]', '[admin_ckey]', '[text]', '[timestamp]', '[server]', INET_ATON(IF('[world.internet_address]' LIKE '', '0', '[world.internet_address]')), '[world.port]', '[GLOB.round_id]','[secret]', [expiry ? "'[expiry]'" : "NULL"])")
+	if(type == "note" && isnull(note_severity))
+		var/list/severity_list = list("High Severity" = 4, "Medium Severity" = 3, "Minor Severity" = 2, "No Severity" = 1)
+		note_severity = input("Set the severity of the note.", "Severity", null, null) as null|anything in severity_list
+		if(!note_severity)
+			return
+		note_severity = severity_list[note_severity] // We want this stored as a number
+		note_severity = sanitizeSQL(note_severity) // This might not be neccesary
```

Comment:
This should be an indentation level up so all the value is always run through sensitization, not just when there's a severity.